### ISSUE TITLE
Pulls Accordion doesn't expand for DSR 

### ISSUE THREAD

**Issue by Shibbywan** [k0etsu/xivodreview#21]
Think there's an issue with the apostrophe, the accordion doesn't open so it's unusuable for DSR. Doesn't work on chrome or firefox

```
selector-engine.js:37 Uncaught DOMException: Failed to execute 'querySelectorAll' on 'Element': '#collapse-Dragonsong'sReprise' is not a valid selector.
    at Object.find (https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/js/bootstrap.bundle.min.js:6:8490)
    at Object.getMultipleElementsFromSelector (https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/js/bootstrap.bundle.min.js:6:9390)
    at HTMLButtonElement.<anonymous> (https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/js/bootstrap.bundle.min.js:6:20856)
    at HTMLDocument.n (https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/js/bootstrap.bundle.min.js:6:3886)

**Comment by k0etsu** [comment]
Can you give me pair of links to test with? I haven't actually done any DSR so it wasn't something I was aware of.

**Comment by Shibbywan** [comment]
https://xivodreview.com?youtube=1KCgu3AXbAo&fflogs=3yPjqwF1Z8bpVYAC&offset=0

**Comment by k0etsu** [comment]
Should be fixed in bb1457e9666b7803c8801310a61c91ab86a743bb and is live
